Dashboard/va.gov.artemis.ui/Views/PatientContact/_CallTab.cshtml

```diff
@@ -86,6 +86,7 @@
                 @if (!Model.IsLast)
                 {
                     <button type="button" id="next-btn" class="btn btn-primary btn-submit" data-nav="@Model.NextTabIndex">Next&nbsp;&nbsp;<span class="glyphicon glyphicon-arrow-right"></span></button>
+                    <button type="button" id="next-btn" class="btn btn-primary btn-submit" data-nav="@Model.LastTabIndex"><span class="glyphicon glyphicon-ok"></span>&nbsp;&nbsp;End</button>
```

Comment:
Suggest giving the button a different id, something like  id="end-btn"

Reply:
agreed.  wish this had a like button.